### PR TITLE
Update task images from `.png` -> `.svg`

### LINKED ISSUE #186: Host benchmark images as svg
Inverting the colors for dark mode doesn't work with pngs. @mvinyard can you replace them with SVGs?

### PR DESCRIPTION
**Pull request by mvinyard**
## Describe your changes
* Updating task images from `.png` to `.svg`

## Issue ticket number and link
Closes #186

## Checklist before requesting a review
- [x] I have performed a self-review of my code

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [ ] New functionality
  - [ ] Major changes
  - [x] Minor changes
  - [ ] Bug fixes

- [x] Proposed changes are described in the CHANGELOG.md

- [x] CI Preview succeeds and looks good!

### CODE REVIEW

File: results/batch_integration_embed/index.qmd [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: "Batch integration embed"
 subtitle: "Removing batch effects while preserving biological variation (embedding output)"
 engine: knitr
-image: featured.png
+image: batch_integration_embed.svg
```
Comment (rcannood) [inline results/batch_integration_embed/index.qmd:5]: Can you rename them all to thumbnail.svg?

```suggestion
image: thumbnail.svg
```

**Review by rcannood — CHANGES_REQUESTED** [review]

File: results/batch_integration_embed/index.qmd (hunk 1 above)
Comment (mvinyard) [reply results/batch_integration_embed/index.qmd:5]: Sure! Meant to ask you about that - thanks

**Review by rcannood — CHANGES_REQUESTED** [review]
A lot of thumbnails are not being rendered properly. See individual comments for more info.

Additionally, please add an entry to the CHANGELOG.md.

After making the suggested changes, please check whether everything looks OK or not. You can do this by running `quarto preview` on your local system, or by pushing your commits and letting the CI render it for you, after which the preview will be available [here](https://update-imgs--openproblems.netlify.app/results/).

**Comment by mvinyard** [comment]
Thanks for pointing this out @rcannood ! My brain was on auto and didn't notice the slight difference in naming within each `index.qmd`. I've committed these fixes per your suggestions and testing the render now. I've also added changes to the CHANGELOG.md. Will update if I have any problems in rendering these images, else will request your review for a final time

File: results/denoising/.DS_Store
Comment (rcannood) [inline results/denoising/.DS_Store:1]: Please remove this :)

File: results/dimensionality_reduction/.DS_Store
Comment (rcannood) [inline results/dimensionality_reduction/.DS_Store:1]: Please remove this :)

File: results/label_projection/.DS_Store
Comment (rcannood) [inline results/label_projection/.DS_Store:1]: Please remove this :)

File: results/matching_modalities/.DS_Store
Comment (rcannood) [inline results/matching_modalities/.DS_Store:1]: Please remove this :)

File: results/spatial_decomposition/.DS_Store
Comment (rcannood) [inline results/spatial_decomposition/.DS_Store:1]: Please remove this :)

File: results/multimodal_data_integration/.DS_Store
Comment (rcannood) [inline results/multimodal_data_integration/.DS_Store:1]: Please remove this :)

File: results/denoising/thumbnail_image.denoising.svg
Comment (rcannood) [inline results/denoising/thumbnail_image.denoising.svg:1]: Please rename this to `thumbnail.svg`

**Review by rcannood — CHANGES_REQUESTED** [review]
No worries, can happen :relaxed: 

Here are more some requested changes.

* Denoising thumbnail image still needs to be renamed
* Please remove .DS_Store files
* Add an entry to the changelog

**Comment by rcannood** [comment]
Took care of the remaining issues :relaxed:

**Review by rcannood — APPROVED** [review]

**Comment by mvinyard** [comment]
Thank you, @rcannood !

**Pull request merged**